willow/app/views/sufia/admin/stats/_deposits.html.erb

```diff
@@ -0,0 +1,18 @@
+<h3>Deposits By Users (<%= @presenter.date_filter_string %>)</h3>
+
+<ul>
+  <% @presenter.depositors.each do |usr| %>
+      <% if usr[:user].present? %>
+        <li>
+          <a href="<%= sufia.profile_path(usr[:key]) %>" title="View user's profile"><%= usr[:user].name %></a>
+          <% num_files = usr[:deposits]%>
+          deposited <%= pluralize(num_files, "file") %>
+        </li>
+      <% else %>
+        <li>
+          <% num_files = usr[:deposits]%>
+          usr[:key] deposited deposited <%= pluralize(num_files, "file") %>
```

Comment:
this line looks wrong? should it be the same as above:
`deposited <%= pluralize(usr[:deposits], "file") %>`